### PR TITLE
[INLONG-591][Doc] Add document for multiple sink of iceberg

### LINKED ISSUE #591: [Feature][Doc] Add document for multiple sink of iceberg
### Description

_No response_

### Use case

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by thesumery**
[INLONG-591][Doc] Add document for multiple sink of iceberg

### Prepare a Pull Request
- Add document for multiple sink of iceberg
- Fixes #591 

### Motivation

*Add document for multiple sink of iceberg.*

### Modifications

*Add document for multiple sink of iceberg.*

### CODE REVIEW

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/load_node/iceberg.md [hunk 1]
```diff
@@ -148,6 +148,98 @@ TODO
...
+  },
+  "table": "products",
+  "ts": 1589373560798,
+  "type": "UPDATE"
+} 
+```
+'topic-pattern' 为 '{database}_${table}', 提取后的 Topic 为 'inventory_products' ('database', 'table' 为元数据字段,
+'id' 为物理字段)
+
+'topic-pattern' 为 '{database}_${table}_${id}', 提取后的 Topic 为 'inventory_products_4' ('database', 'table'
```
Comment (EMsnap) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/load_node/iceberg.md:225]: 111 ?

File: docs/data_node/load_node/iceberg.md [hunk 2]
```diff
@@ -147,6 +147,103 @@ TODO
 ### Usage for InLong Manager Client
 TODO
 
+## Feature
+### Multiple table sink
+Currently Iceberg support multiple table sinking, it require FLINK SQL create table parameters add  
+`'sink.multiple.enable' = 'true'` and target table schema can only be defined as `BINARY` or `STRING`
```
Comment (gong) [inline docs/data_node/load_node/iceberg.md:153]: `BINARY` should change to `BYTES`

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/load_node/iceberg.md [hunk 3]
```diff
@@ -148,6 +148,98 @@ TODO
 ### InLong Manager Client 用法
 TODO
 
+## 特征
+### 多表写入
+目前 Iceberg 支持多表同时写入，需要在 FLINK SQL 的建表参数上添加 `'sink.multiple.enable' = 'true'` 并且目标表的schema
+只能定义成 `BINARY` 或者 `STRING` ，以下是一个建表语句举例：
+```
```
Comment (gong) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/load_node/iceberg.md:155]: `BINARY` should be `BYTES`

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/load_node/iceberg.md (hunk 1 above)
Comment (EMsnap) [reply i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/load_node/iceberg.md:225]: 111 -> 4

**Review by gong — APPROVED** [review]

**Review by EMsnap — APPROVED** [review]

**Pull request merged**